tests/conftest.py

```diff
@@ -2391,6 +2394,7 @@ def javasdk_pod_fixture(request, scope_name):
     javas3_pod_name = create_unique_resource_name(constants.JAVAS3_POD_NAME, scope_name)
     javas3_pod_dict["metadata"]["name"] = javas3_pod_name
     update_container_with_mirrored_image(javas3_pod_dict)
+    update_container_with_proxy_env(javas3_pod_dict)
```

Comment:
lets use `skipif_proxy_cluster` to skip the relevant tests instead of this @dahorak

Reply:
Skipped the `TestS3WithJavaSDK` tests in the commit [9ad4e37](https://github.com/red-hat-storage/ocs-ci/pull/8301/commits/9ad4e37e09d34c26752d8998c3cdaf6e211597e3). I would propose to leave the `update_container_with_proxy_env` here, to keep it consistent and also it will be required if we will want to find the proper solution for the issue.